apps/src/codebridge/components/ResizeBar.tsx

```diff
@@ -11,37 +10,63 @@ interface ResizeBarProps {
   separatorProps: SeparatorProps;
 }
 
-export const RESIZE_BAR_SIZE_PX = 13;
+export const RESIZE_BAR_SIZE_PX = 1;
 
+// A resize bar that can be dragged to resize two adjacent panels.
+// The bar is 1px wide, but when it is hovered/focused or being dragged, it
+// becomes 5px wide. The area that can be grabbed is also 5px wide.
+// The resize bar should be used with useResizable from react-resizable-layout.
 const ResizeBar: React.FunctionComponent<ResizeBarProps> = ({
```

Comment:
Could this be a `lab2` component so that Lab2 apps other than `codebridge` can also use it?

Reply:
moved!